client/packages/coldchain/src/Equipment/DetailView/Tabs/Summary.tsx

```diff
@@ -184,35 +186,37 @@ export const Summary = ({ draft, onChange, locations }: SummaryProps) => {
             />
           </Row>
         </Section>
-        <Section heading={t('heading.cold-chain')}>
-          <Row label={t('label.cold-storage-location')}>
-            {locations ? (
-              <AutocompleteMulti
-                isOptionEqualToValue={(option, value) =>
-                  option.value === value.value
-                }
-                defaultValue={defaultLocations}
-                filterSelectedOptions
-                getOptionLabel={option => option.label}
-                inputProps={{ fullWidth: true }}
-                onChange={(
-                  _event,
-                  newSelectedLocations: {
-                    label: string;
-                    value: string;
-                  }[]
-                ) => {
-                  onChange({
-                    locationIds: ArrayUtils.dedupe(
-                      newSelectedLocations.map(location => location.value)
-                    ),
-                  });
-                }}
-                options={locations}
-              />
-            ) : null}
-          </Row>
-        </Section>
+        {(!isCentralServer || draft.storeId == storeId) && (
```

Comment:
`unless they relate to the currently logged in store`

So draft.storeId != storeID?

Reply:
The logic is reversed, so we show the section if.
A) we are not on central server
or
B) If we are logged into the same store

Does that make sense? If it not working that way for you?

Reply:
Oh sorry my brain died for a moment. Yup it makes sense